problem-set-2/Praktikum 2.sql

```diff
@@ -0,0 +1,33 @@
+-- Nama : Ilham Kurniawan
+-- Nim : H071231024 
+
+-- No 1
+SELECT productcode AS 'Kode Produk', productname AS 'Nama Produk', quantityinstock AS 'Jumlah Produk'
+FROM products
+WHERE quantityinstock BETWEEN 5000 AND 6000
+ORDER BY quantityinstock ASC;
+
+-- No 2
+SELECT ordernumber AS 'Nomor Pesanan', orderdate AS 'Tanggal Pesanan', `status`, customernumber AS 'Nomor Pelanggan'
+FROM orders
+WHERE `status` IN ("In Process", "On Hold", "Disputed", "Cancelled", "Resolved")
```

Comment:
Operator IN belum diajarkan. Selain itu, cara paling pendeknya cukup menggunakan operator '!='.